src/masquerade/providers/web_api.py

```diff
@@ -103,7 +103,7 @@ def make_request(address, zone, out_spatial_reference, max_locations):
     headers = {'Referer': 'https://masquerade.agrc.utah.gov'}
     url = f'{WEB_API_URL}/{_cleanse_street(address)}/{_cleanse_zone(zone)}'
 
-    response = session.get(url, params=parameters, headers=headers)
+    response = session.get(url, params=parameters, headers=headers, timeout=30)
```

Comment:
might i suggest something like 5 or 10?

Reply:
I'll switch it to 10. I wasn't sure what a good timeout would be.